docs/make.bat

```diff
@@ -18,6 +18,7 @@ if "%1" == "help" (
 	echo.Please use `make ^<target^>` where ^<target^> is one of
 	echo.  html       to make standalone HTML files
 	echo.  livehtml   to make live HTML files to view the docs on a local web server
+	echo.  dockerhtml to make live HTML files and view through a local docker container
```

Comment:
Definitely worth clarifying here.

Reply:
Used solely to bind sphinx to the Docker Container - to do so requires deliberate --host "0.0.0.0" - will add a comment